.github/workflows/terraform.yml

```
@@ -0,0 +1,139 @@
+  issues: write
+
+jobs:
+  terraform:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: install jq
+        run: sudo apt-get install jq
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)